src/main/java/Duke.java

```diff
@@ -19,26 +25,76 @@ public static void main(String[] args) {
 //                    "\n____________________________________________________________\n";
 //            System.out.println(output);
 //            input = sc.nextLine();
-//        } Level-1
-        String[] myList;
-        myList = new String[100];
+//        }
+//        Level-1
+//        String[] myList;
+//        myList = new String[100];
+//        Scanner sc = new Scanner(System.in);
+//        String input = sc.nextLine();
+//        int numberOfInputs = 0;
+//        while(!input.equals("list")){
+//            myList[numberOfInputs] = input;
+//            numberOfInputs++;
+//            String output = "____________________________________________________________\n " +
+//                    "Added: " +
+//                    input +
+//                    "\n____________________________________________________________\n";
+//            System.out.println(output);
+//            input = sc.nextLine();
+//        }
+//        for(int i = 0; i < numberOfInputs; i++){
+//            System.out.print(i + 1);
+//            System.out.println(". " + myList[i]);
+//        }
+//        Level-2
```

Comment:
Useless comments. All comments should be written in English and Comments should be indented relative to their position in the code. Redundant code from previous versions should be deleted so as to keep the code tidy.